storagemanager/storage/views.py

```diff
@@ -23,49 +23,65 @@ class ShowAllListView(ListView):
     model = Item
     template_name = 'storage/show_all.html'
     context_object_name = 'all_items'
-    ordering = ['category', 'type']
+    ordering = ['category', 'type', 'model']
 
 def show_items_from_category(request):
     categories_set = set()                      
     for item in Item.objects.all():
         categories_set.add(item.category)
-    sorted_items = sorted(Item.objects.filter(category=request.POST.get('category')), key=attrgetter('type', 'model'))
+    sorted_items = Item.objects.filter(category=request.POST.get('category')).order_by('type', 'model')
```

Comment:
I'd put `request.POST.get('category')` into variable above as it makes reading this line a bit hard.